src/atoms/Icon/constants.js

```diff
@@ -163,6 +165,7 @@ module.exports = {
     'lock',
     'lockBlue',
     'lockOrange',
+    'lunchPail',
```

Comment:
LOL I think this is a briefcase but I like lunchpail

Reply:
we could call it `rectangleWithHandle` to keep it generic? 😂

Reply:
![Screen Shot 2020-12-30 at 11 26 12 AM](https://user-images.githubusercontent.com/16748392/103366950-dd648d80-4a91-11eb-9dde-2681744cbdd1.png)
I just used the name of the image in Figma